packages/amplify-e2e-tests/src/__tests__/delete.test.ts

```diff
@@ -86,3 +85,20 @@ async function bucketExists(bucket: string) {
     throw error;
   }
 }
+
+async function bucketNotExists(bucket: string) {
+  const s3 = new S3();
+  const params = {
+    Bucket: bucket,
+    $waiter: { maxAttempts: 10 },
+  };
+  try {
+    await s3.waitFor('bucketNotExists', params).promise();
+    return true;
+  } catch (error) {
+    if (error.statusCode === 200) {
```

Comment:
Maybe return false after checking the bucket exists? and that the 10 tries did not fail for some other reason